### ISSUE TITLE
`npm init solid app <app-name>` fails

### ISSUE THREAD

**Issue by hexagon6** [ryansolid/solid#30]
```bash
npm init solid app my-solid-app
npx: installed 91 in 7.056s

Creating a new Solid app in /home/USER/tmp/my-solid-app.

internal/modules/cjs/loader.js:584
    throw err;
    ^

Error: Cannot find module './yarn.lock.cached'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.resolve (internal/modules/cjs/helpers.js:30:19)
    at createApp (/home/USER/.npm/_npx/23726/lib/node_modules/create-solid/createSolid.js:254:17)
    at Object.<anonymous> (/home/USER/.npm/_npx/23726/lib/node_modules/create-solid/createSolid.js:169:1)
    at Module._compile (internal/modules/cjs/loader.js:701:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:712:10)
    at Module.load (internal/modules/cjs/loader.js:600:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:539:12)
    at Function.Module._load (internal/modules/cjs/loader.js:531:3)
    at Module.require (internal/modules/cjs/loader.js:637:17)
```

npm/node is installed with nvm

**Comment by ryansolid** [comment]
Thanks for reporting this. I have used this to bootstrap a couple apps now but it is still pretty early stages. Do you have yarn installed? I haven't been using yarn to do this, but I believe create-react-app this is based on prefers yarn where possible, and I might have missed something on the yarn side.

**Comment by hexagon6** [comment]
@ryansolid yarn@1.9.4 is globally installed.

**Comment by hexagon6** [comment]
@ryansolid 
https://github.com/ryansolid/create-solid/blob/master/createSolid.js#L208
`const useYarn = useNpm ? false : shouldUseYarn();` 
maybe determine `useNpm` not only on the passed command-parameter, but on the caller if possible.

**Comment by hexagon6** [comment]
for example:  
`process._` gives back: 
* with npm: '/home/USER/.nvm/versions/node/v10.15.3/bin/npm'
* with yarn '/home/USER/.yarn/bin/yarn',

**Comment by hexagon6** [comment]
@ryansolid basically do a `process._.endsWith('npm')` or `process._.endsWidth('yarn')` to check

**Comment by ryansolid** [comment]
Sorry took me a few days to get to. I was in the middle of some last critical fixing bugs for the new 0.7.0 release.

I did what you suggested to detect (process.env._). I also put the onus on yarn to prove it should be used, so npm is the default now (for some weird scenario I can't predict). Finally I also fixed the yarn problem that was preventing it from working. Let me know if you have any further issues.
